http/jsonapi/runtime/parameters_test.go

```diff
@@ -9,6 +9,63 @@ import (
 	"testing"
 )
 
+func TestScanStringParametersInQuery(t *testing.T) {
+	tests := []struct {
+		path   string
+		result string
+	}{
+		{"/?q=", ""},
+		{"/?q=Super", "Super"},
+		{"/?q=Super%20Plus", "Super Plus"},
```

Comment:
Maybe add case for `Super Plus` without any encoding.

Reply:
That would be invalid HTTP